### PR TITLE
To-Do Task

### PR DESCRIPTION
**Pull request by altontonn**
### To-Do Task
 - added index, and description and completed  to my class constructor
 - worked on checkboxes to select all items upon delete
 - added event listener to the checkboxes to strike through the items

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -29,4 +29,4 @@ <h3 id="clear">Clear All Completed</h3>
             </div>
         </div>
     </div>
-</body>
\ No newline at end of file
+</body>
```
Comment (wuletawwonte) [inline src/index.html:32]: - [ ] There is no file import with methods related to status updates(completed: true/false). Please add a new javascript file and put status update methods in it. Then import it as a JS module.

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @altontonn  👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.html (hunk 1 above)
Comment (altontonn) [reply src/index.html:32]: Hi @wuletawwonte I do not understand, kindly come again

Comment (wuletawwonte) [reply src/index.html:32]: Hi @altontonn 
The first requirement is not implemented. I think you are supposed to create a new javascript file and write your status update methods(Checkbox status change handler methods) in it. Then import it as a module in the index.js file.

**Review by mwanawabangona — APPROVED** [review]
Hi @altontonn  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**